hoodie-hive/src/main/java/com/uber/hoodie/hive/HiveSyncConfig.java

```diff
@@ -21,30 +21,45 @@
 import com.beust.jcommander.Parameter;
 
 import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Configs needed to sync data into Hive.
  */
```

Comment:
Github PR is doing this fancy thing where it is hiding the lines - you will find a small blue icon on the left which when expanded shows the license. Just checked - All these files have valid licenses in them.